po/da.po

```diff
@@ -2002,7 +2025,7 @@ msgstr ""
 
 #: src/welcome-day-1/what-is-rust.md:21
 msgid "Rust fits in the same area as C++:"
-msgstr ""
+msgstr "Rust passer ind i det samme område som C++:"
```

Comment:
What do you think about something like "Rust har det samme anvendelsesområde som C++? It wasn't immediately clear what "område" was referring to without looking at the original text.

Reply:
That is much better, thanks!